### DISCUSSION TITLE
Wire payment module - incorrect wording

### DISCUSSION THREAD

**Discussion by Caleydon** [PrestaShop/PrestaShop#37051] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The module [ps_wirepayment](https://github.com/PrestaShop/ps_wirepayment) is incorrectly assuming that purchased products are always a physical goods. That force the shop owner develop and fix the module when he is selling services or digital goods.

### Expected behavior

The wording in the module by default should be universal, accommodating the sale of physical products, as well as services and digital goods.

### Steps to reproduce

The are 2 options how to fix it:

1. Change the meaning of the following sentences to make them more universally applicable.

- Goods will be reserved %s days for you and we'll process the order immediately after receiving the payment.
- Goods will be reserved %s day for you and we'll process the order immediately after receiving the payment.
- Your order will be sent as soon as we receive payment.
- in the email notification template: You can expect delivery as soon as your payment is received.

2. Add logic into the module that detects the type of goods in the shopping cart/ordered and dynamically offers sentences with the correct meaning based on the type of goods (physical products, services, or digital goods or mixture of the types).

### PrestaShop version(s) where the bug happened

8.2.0

### If your bug is related to a module, specify its name and its version

ps_wirepayment v2.2.0

**Comment by Hlavtox** [comment]
Hi @Caleydon, I converted this issue to the Idea since it fits better here.
